main.py

```diff
@@ -97,6 +114,10 @@ def __show_help_screen(self, signal_emitter=None):
     def __show_prev_screen(self, signal_emitter=None):
         self.__set_overlay(self.prev)
 
+    def show_game_over_screen(self, signal_emitter=None):
```

Comment:
this should be a private method, use a signal to trigger it

Reply:
i tried it didnt work and whats the problem with it being public if i make it private i have to add so much more code for it to be able to work

Reply:
Because it follows the architecture of the rest of the code. By looking at the Controller we can grasp how what signals lead to what screens. If one of the screens calls a screen change by itself, this is not visible from looking at the controller code and the flow becomes harder to comprehend and more prone to bugs.

Reply:
alright i was able to do that